skore/src/skore/sklearn/_estimator/metrics_accessor.py

```diff
@@ -371,6 +371,8 @@ def report_metrics(
                 results.columns = flatten_multi_index(results.columns)
             if isinstance(results.index, pd.MultiIndex):
                 results.index = flatten_multi_index(results.index)
+            if isinstance(results.index, pd.Index):
+                results.index = results.index.str.replace(r"_?\(s\)$", "_s", regex=True)
```

Comment:
```suggestion
            if isinstance(results.index, pd.Index):
                results.index = results.index.str.replace(
                    r"\((.*)\)$", r"\1", regex=True
                )
```
Feels a bit cleaner/scalable to me

Reply:
Should this be consistent across other `metrics_accessor` files too? 

fyi, with the suggested regex, the test no longer expects separate `predict_time_train_s` and `predict_time_test_s` metrics, but instead expects a single `predict_time_s` metric...

Reply:
Oh sorry, my bad, didn't update the tests after the suggested removal of the following lines:
 ```diff   
-   timings = report.metrics.timings()
-   assert isinstance(timings, pd.DataFrame)

-   actual_index = timings.index.tolist()
-   assert "Fit time (s)" in actual_index
-   assert "Predict time train (s)" in actual_index
-   assert "Predict time test (s)" in actual_index
```

Reply:
So this list was looking for the `predict_time_test_s` and `predict_time_test_s` labels:
```python
assert results.index.tolist() == [
        "precision_0",
        "precision_1",
        "recall_0",
        "recall_1",
        "roc_auc",
        "brier_score",
        "fit_time_s",
        "predict_time_s",
    ]
```

Reply:
Im not sure I get it but is this resolved? If so please resolve